style.css

```diff
@@ -0,0 +1,489 @@
+}
+
+.most__cards {
+  border-top: 1px solid black;
+  display: flex;
+  flex: 0.85;
+  flex-direction: column;
+  margin-left: auto;
+}
+
+.most__cards > h5 {
+  margin-top: 10px;
+  font-weight: 800;
+  font-size: 14px;
+}
+
+.card__info > img {
+  height: 120px;
+  cursor: pointer;
+  width: 200px;
+}
+
+.card__infoBox {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.most__popular {
+  display: flex;
+  margin-left: 20px;
+  flex-direction: column;
+}
+
+.most__popularTitle {
+  border-top: 1px solid black;
+  width: 300px;
+}
+
+.most__popularTitle > h5 {
+  margin-top: 10px;
+  font-weight: 800;
+  font-size: 14px;
+}
+
+.most__popularInfo {
+  width: 50%;
+}
+
+.most__popularInfo > p:hover {
+  color: grey;
+}
+
+.footer__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-top: 1px solid lightgray;
+}
+
+.times__logo {
+  display: flex;
+  margin-top: 20px;
+  justify-content: center;
+  margin-bottom: 20px;
+}
+
+.footer__title {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.title__columnInfo > p:hover {
+  text-decoration: underline;
+}
+
+.title__columnSub {
+  border-left: 1px solid lightgrey;
+  padding-left: 10px;
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 1009px) {
+  .img__box > img {
+    width: 100vw;
+    justify-content: center;
+  }
+
+  .user {
+    display: flex;
+    font-size: 20px;
+  }
+
+  .right__side > button {
+    display: none;
+  }
+
+  .header__buttons {
+    display: flex;
+    background-color: white;
+    justify-content: flex-end;
+  }
+
+  .header__buttons > button {
+    background: transparent;
+    outline: none;
+    border: none;
+    color: grey;
+    margin-right: 10px;
+    cursor: pointer;
+    text-transform: uppercase;
+  }
+
+  .header__text {
+    display: none;
+  }
+
+  .left__icon {
+    display: none;
+  }
+
+  .comment > p {
+    width: 80%;
+    color: gray;
+  }
+
+  .editor__pick {
+    display: none;
+  }
+
+  .times__logo {
+    display: flex;
+  }
+
+  .footer__title {
+    display: flex;
+    flex-direction: column;
+    margin-left: 10px;
+  }
+
+  .title__column {
+    cursor: pointer;
+    font-size: 20px;
+    margin-top: 10px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    border-top: 1px solid lightgray;
+  }
+
+  .title__columnInfo {
+    display: none;
+  }
+
+  .title__columnSub {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 950px) {
+  .img__box > img {
+    width: 100vw;
+    justify-content: center;
+  }
+
+  .comment > p {
+    width: 80%;
+    color: gray;
+  }
+
+  .editor__pick {
+    display: none;
+  }
+}
+
+@media only screen and (max-width: 760px) {
+  .img__box > img {
+    width: 100vw;
+    justify-content: center;
+  }
+
+  .editor__pick {
+    display: none;
+  }
+
+  .advert__box {
+    display: none;
+  }
+
+  .comment > p {
+    width: 100%;
+    color: gray;
+  }
+
+  .most__popular {
+    display: none;
+  }
```

Comment:
- [ ] This could be simplified, look for patterns similar to this, and change or create a helper CSS Class
```css
BAD
 .editor__pick {
    display: none;
  }

  .advert__box {
    display: none;
  }

  .most__popular {
    display: none;
  }

GOOD

 .editor__pick,
 .advert__box,
  .most__popular {
    display: none;
  }
```